Lib/test/test_pdb.py

```diff
@@ -1857,7 +1858,9 @@ def test_errors_in_command(self):
             "((Pdb)) *** NameError: name 'doesnotexist' is not defined",
             'LEAVING RECURSIVE DEBUGGER',
             '(Pdb) ',
-        ])
+        ]
+        self.assertTrue(all(e == s for (e, s) in zip(stdout_lines, expected_lines)))
```

Comment:
This could use a comment explaining why we aren't checking for an exact match.

Also, the use of assertTrue means we won't get a detailed error if the test fails. Perhaps you could loop over the pairs of lines and `self.assertEqual` for each line instead.